web/styles/left_sidebar.css

```diff
@@ -1365,29 +1366,21 @@ li.topic-list-item {
     }
 
     #hide-more-direct-messages {
-        display: block;
+        grid-template-columns: inherit;
```

Comment:
I don't understand why there's a need to inherit or talk about grid-columns here. Referencing the `grid-area` for the grid definition above should suffice, no?

Reply:
For each of the selector (e.g. `.dm-box`, `.subscription_block`) sharing the grid definition, there is a grid-template-columns defined explicitly for each of them, e.g. [link](https://github.com/zulip/zulip/blob/79f858b4b8253c59d5fec1803431f0ea7c2447fa/web/styles/left_sidebar.css#L1090). In this case, since `#hide-more-direct-messages` is a child of `#direct-messages-section-header`, we don't need to define `grid-template-columns` explicitly and just inherit the values from the parent container.

Reply:
This seems acceptable to me but @karlstolley can follow up next week if he thinks we can do this more cleanly.